File: WEEK1/DAY3/Prog3/P3.java
import java.util.*;
public class P3 {
    public static void multiple15(int N){
        for(int i=1; i<=N; i++){
            if(i%15==0){
                System.err.println(i);
            }
        }
    }
    public static void main(String[] args) {
        Scanner sc=new Scanner(System.in);
        System.out.println("Print all numbers 1→N divisible by 3 and 5");
        System.out.print("Enter N: ");
        int N=sc.nextInt();
        multiple15(N);
        sc.close();
    }
}

File: WEEK1/DAY3/Prog3/P3.c
#include <stdio.h>
void multiple15(int N){
    for(int i=1;i<=N;i++){
        if(i%15==0){
            printf("%d\n",i);
        }
    }
}
int main(){
    int N;
    printf("Print all numbers 1→N divisible by 3 and 5\n");
    printf("Enter N: ");
    scanf("%d",&N);
    multiple15(N);
}